src/EnergyPlus/UnitarySystem.hh

```diff
@@ -287,9 +286,6 @@ namespace UnitarySystems {
         int m_HeatRecoveryOutletNodeNum;
         int m_DesignSpecMSHPIndex;
         Real64 m_NoLoadAirFlowRateRatio;
-        Real64 m_IdleMassFlowRate;
-        Real64 m_IdleVolumeAirRate; // idle air flow rate [m3/s]
-        Real64 m_IdleSpeedRatio;
```

Comment:
I have wanted to get rid of these for a while now. When a new developer comes in they create their own new model variables instead of using existing, when the existing would work just as well. Running into the memory issue I had gave me the incentive. Idle flow rate is the same as NoLoad or NoCoolHeat variables so I just substituted and eliminated these.